pkg/services/oauthtoken/oauth_token_util.go

```diff
@@ -8,15 +8,28 @@ import (
 	"github.com/grafana/grafana/pkg/infra/log"
 	"github.com/grafana/grafana/pkg/login/social"
 	"github.com/grafana/grafana/pkg/models"
+	"github.com/grafana/grafana/pkg/registry"
 	"golang.org/x/oauth2"
 )
 
 var (
 	logger = log.New("oauthtoken")
 )
 
+func init() {
+	registry.RegisterService(&OAuthTokenService{})
+}
+
+type OAuthTokenService struct {
```

Comment:
Since the package is already named `oauthtoken` this could be named `Service`. Name of the file could be renamed to reflect it's now service rather than a util.